src/helpers/permission/permission-helper.js

```diff
@@ -2,8 +2,8 @@ import { getPermissionApi } from '../shared/user-login';
 
 const accessApi = getPermissionApi();
 
-export async function listPermissions(limit, offset, orderBy, application, resourceType, verb, permission, options) {
-  return await accessApi.listPermissions(limit, offset, orderBy, application, resourceType, verb, permission, options);
+export async function listPermissions(limit, offset, orderBy, application, resourceType, verb, permission, excludeGlobals, excludeRoles, options) {
```

Comment:
I see a change in the order of the arguments (options are now pushed back), but no client upgrade? Does this mean that the options object was never used or was the order incorrect all along?

Reply:
Oh, I forgot to commit the client dependency update :facepalm:
Good catch, thank you @Hyperkid123